test/system/bans_test.rb

```diff
@@ -91,20 +91,35 @@ def test_banned_user_can_see_the_ban_notification_for_one_domain
 
   def test_banned_user_can_see_the_ban_notification_for_two_domains
     Ban.create!(user: @participant,
-                domain_name: 'temporary.test',
+                domain_name: @valid_auction_with_offers.domain_name,
                 valid_from: Time.zone.today - 1, valid_until: Time.zone.today + 5)
 
     sign_in_manually
 
     text = <<~TEXT.squish
       You are banned from participating in auctions for
-      domain(s): temporary.test, no-offers.test.
+      domain(s): with-offers.test, no-offers.test.
     TEXT
 
     visit auctions_path
     assert(page.has_css?('div.ban', text: text))
   end
 
+  def test_banned_user_can_see_the_ban_notification_only_for_active_auctions
+    Ban.create!(user: @participant,
+                domain_name: 'dummy-domain.net',
+                valid_from: Time.zone.today - 1, valid_until: Time.zone.today + 5)
```

Comment:
Magic numbers. The intention is not clear. Why just 5? You might need to use `travel_to`, otherwise sporadic errors might appear due to variable current time.

Reply:
We are already traveling to in the setup phase, these numbers are just to get a range. Moved them to named constants anyway.